modules/openxBidAdapter.js

```diff
@@ -107,9 +107,11 @@ const converter = ortbConverter({
       fledgeAuctionConfigs = Object.entries(fledgeAuctionConfigs).map(([bidId, cfg]) => {
         return {
           bidId,
-          config: Object.assign({
-            auctionSignals: {},
-          }, cfg)
+          config: mergeDeep(Object.assign({}, cfg), {
+            auctionSignals: {
+              ortb2Imp: context.impContext[bidId]?.ortbRequest?.imp?.[0],
```

Comment:
you can use `context.impContext[bidId]?.imp` to get the precise imp tied to the bid id (instead of the first in the request)